com_migratetojoomla/admin/forms/main.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<form>
+	<field
+		name="mediaoptions"
+		type="list"
+		label="COM_MIGRATETOJOOMLA_SELECT_OPTION_FOR_MEDIA_LABEL"
+		description="COM_MIGRATETOJOOMLA_SELECT_OPTION_FOR_MEDIA_DESC"
+		class="form-select"
+		default="1"
+		validate="options"
+	>
+		<option value="http">COM_MIGRATETOJOOMLA_HTTP</option>
+		<option value="fs">COM_MIGRATETOJOOMLA_FS</option>
+		<option value="ftp">COM_MIGRATETOJOOMLA_FTP</option>
+	</field>
+	<field
+		name="livewebsiteurl"
+		type="text"
+		showon="mediaoptions:http"
+		label="COM_MIGRATETOJOOMLA_HTTP_LINK_LABEL"
+		description="COM_MIGRATETOJOOMLA_HTTP_LINK_DESC"/>
+	<field
+		name="basedir"
+		type="text"
+		showon="mediaoptions:fs"
+		label="COM_MIGRATETOJOOMLA_WEBSITE_BASE_DIRECTORY_LABEL"
+		description="COM_MIGRATETOJOOMLA_WEBSITE_BASE_DIRECTORY_DESC"/>
+	<field
+		name="ftphost"
+		type="text"
+		showon="mediaoptions:ftp"
+		label="COM_MIGRATETOJOOMLA_FTP_HOST_LABEL"/>
+	<field
+		name="ftpport"
+		type="int"
+		showon="mediaoptions:ftp"
+		label="COM_MIGRATETOJOOMLA_FTP_PORT_LABEL"/>
+	<field
+		name="ftpusername"
+		type="text"
+		showon="mediaoptions:ftp"
+		label="COM_MIGRATETOJOOMLA_FTP_USERNAME_LABEL"/>
+	<field
+		name="ftppassword"
+		type="text"
+		showon="mediaoptions:ftp"
+		label="COM_MIGRATETOJOOMLA_FTP_PASSWORD_LABEL"/>
+	<field
+		name="ftpprotocoloptions"
+		type="list"
+		label="COM_MIGRATETOJOOMLA_FTP_PROTOCOL_LABEL"
+		class="form-select"
+		default="1"
+		showon="mediaoption:ftp"
+		validate="options"
+	>
+		<option value="1">COM_MIGRATETOJOOMLA_FTP</option>
+		<option value="2">COM_MIGRATETOJOOMLA_FTPS</option>
+		<option value="3">COM_MIGRATETOJOOMLA_SFTP</option>
```

Comment:
If you use strings as values, like you did for mediaoptions, the code later on will be easier to understand.

Reply:
done